Makefile

```diff
@@ -403,6 +404,18 @@ CRDOC = $(shell pwd)/bin/crdoc
 crdoc: ## Download crdoc locally if necessary.
 	$(call go-get-tool,$(CRDOC), fybrik.io/crdoc,v0.5.2)
 
+.PHONY: chainsaw
+chainsaw: ## Find or download chainsaw
+ifeq (, $(shell which chainsaw))
+	@{ \
+	set -e ;\
+	go install github.com/kyverno/chainsaw@v0.1.4 ;\
+	}
```

Comment:
We have a small utility to make this nicer.
```suggestion
$(call go-get-tool,$(CHAINSAW), github.com/kyverno/chainsaw,v0.1.4)
```

Can we also move the version to its own variable, like the other tools?

Reply:
👍

Reply:
Follow up PR if it's ok.